providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/EventStreamObserver.java

```diff
@@ -0,0 +1,101 @@
+package dev.openfeature.contrib.providers.flagd;
+
+import java.util.Map;
+import io.grpc.stub.StreamObserver;
+import dev.openfeature.flagd.grpc.Schema.EventStreamResponse;
+import dev.openfeature.sdk.ProviderEvaluation;
+import com.google.protobuf.Value;
+
+/**
+ * EventStreamObserver handles events emitted by flagd.
+ */
+public class EventStreamObserver implements StreamObserver<EventStreamResponse> {
```

Comment:
Let's annotate with lombo `@Slf4j` (refer - [1] ) .  This will expose a logger which can be accessed with `log.<level>`




[1] https://github.com/open-feature/java-sdk-contrib/blob/main/providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdProvider.java#L41

Reply:
Ah thanks, wasn't sure on the best logging practice here :) Done